packages/application-shell/src/test-utils/test-utils.js

```diff
@@ -60,6 +62,16 @@ const defaultPermissions = { canManageProject: true };
 const mergeOptional = (defaultValue, value) =>
   value === null ? undefined : { ...defaultValue, ...value };
 
+const ApolloProvider = props =>
+  props.mocks ? (
+    <ApolloMockProvider {...props} />
+  ) : (
+    <ApolloActualProvider client={createApolloClient()}>
```

Comment:
So this creates a new apollo client each time it (re)renders?

Reply:
Exactly.